Author: Alessio Treglia <alessio@debian.org>
Description:
 Set installation prefix to /usr (Debian-specific)
 Set GConf schemas's file extension properly.
 Don't install documentation files by default (Debian-specific).
Forwarded: yes
Last-Update: 2010-11-25
---
 Makefile |   18 ++++++++++--------
 1 file changed, 10 insertions(+), 8 deletions(-)

--- gmtp.orig/Makefile
+++ gmtp/Makefile
@@ -1,7 +1,7 @@
 # gMTP Sync tool
 
 PKG_NAME = gmtp
-PREFIX = /usr/local
+PREFIX = /usr
 VER = 0.7
 # Note: If you update above, please update the config.h and pkginfo file as well.
 
@@ -51,23 +51,25 @@ install: gmtp
 	$(INSTALL) -d $(DESTDIR)$(PREFIX)/share/$(PKG_NAME)
 	$(INSTALL) -d $(DESTDIR)$(PREFIX)/share/applications
 	$(INSTALL) -d $(DESTDIR)$(PREFIX)/share/pixmaps
-	$(INSTALL) -d $(DESTDIR)$(PREFIX)/share/doc
-	$(INSTALL) -d $(DESTDIR)$(PREFIX)/share/doc/$(PKG_NAME)
 	$(INSTALL) -d $(DESTDIR)$(PREFIX)/share/gconf
 	$(INSTALL) -d $(DESTDIR)$(PREFIX)/share/gconf/schemas
 	$(INSTALL) -m 755 gmtp $(DESTDIR)$(PREFIX)/bin
 	$(INSTALL) -m 644 icon.png $(DESTDIR)$(PREFIX)/share/$(PKG_NAME)
 	$(INSTALL) -m 644 icon-16.png $(DESTDIR)$(PREFIX)/share/$(PKG_NAME)
 	$(INSTALL) -m 644 stock-about-16.png $(DESTDIR)$(PREFIX)/share/$(PKG_NAME)
-	$(INSTALL) -m 644 README $(DESTDIR)$(PREFIX)/share/doc/$(PKG_NAME)
-	$(INSTALL) -m 644 COPYING $(DESTDIR)$(PREFIX)/share/doc/$(PKG_NAME)
-	$(INSTALL) -m 644 ChangeLog $(DESTDIR)$(PREFIX)/share/doc/$(PKG_NAME)
-	$(INSTALL) -m 644 AUTHORS $(DESTDIR)$(PREFIX)/share/doc/$(PKG_NAME)
 	$(INSTALL) -m 644 gMTP.desktop $(DESTDIR)$(PREFIX)/share/applications
 	$(INSTALL) -m 644 icon.png $(DESTDIR)$(PREFIX)/share/pixmaps
 	$(INSTALL) -m 644 gMTP.schema $(DESTDIR)$(PREFIX)/share/gconf/schemas
 	mv $(DESTDIR)$(PREFIX)/share/pixmaps/icon.png $(DESTDIR)$(PREFIX)/share/pixmaps/gMTPicon.png
-	GCONF_CONFIG_SOURCE=`$(GCONFTOOL) --get-default-source` $(GCONFTOOL) --makefile-install-rule gMTP.schema
+	mv $(DESTDIR)$(PREFIX)/share/gconf/schemas/gMTP.schema $(DESTDIR)$(PREFIX)/share/gconf/schemas/gmtp.schemas
+
+install-doc:
+	$(INSTALL) -d $(DESTDIR)$(PREFIX)/share/doc
+	$(INSTALL) -d $(DESTDIR)$(PREFIX)/share/doc/$(PKG_NAME)
+	$(INSTALL) -m 644 README $(DESTDIR)$(PREFIX)/share/doc/$(PKG_NAME)
+	$(INSTALL) -m 644 COPYING $(DESTDIR)$(PREFIX)/share/doc/$(PKG_NAME)
+	$(INSTALL) -m 644 ChangeLog $(DESTDIR)$(PREFIX)/share/doc/$(PKG_NAME)
+	$(INSTALL) -m 644 AUTHORS $(DESTDIR)$(PREFIX)/share/doc/$(PKG_NAME)
 
 clean:
 	rm -f $(objects) core gmtp *.o *~ $(PKGFILE).gz
